tester/include/spdlog/async.h

```diff
@@ -0,0 +1,87 @@
+
+//
+// Copyright(c) 2018 Gabi Melman.
```

Comment:
If you wanna to use logger lets use the Boost Logger: https://github.com/trunksBT/Almag_RetDriverSimulator/blob/master/src/Utils/Logger.hpp, https://github.com/trunksBT/Almag_RetDriverSimulator/blob/master/src/main.cpp, https://github.com/trunksBT/Almag_RetDriverSimulator/blob/master/CMakeLists.txt . Thanks that you can receive logging onto standard output and/or to the textFile with filtered severity

Reply:
You can observe it here https://github.com/trunksBT/Almag_RetDriverSimulator/blob/master/src/CommandPattern/AlmagController.cpp ( I know that in whole project i broke some rules about operator[] etc... I didn't receive the review too xD )